.github/workflows/workflow.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.7', '3.9']
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
What's the purpose? Is there actually submodules being checked out? If so then it should stay but if not this can likely be removed.

Reply:
I copied from previous templates. No it is not required.